### PR TITLE
Clean up snippets stage deploy yaml

### PR DESCRIPTION
**Pull request by jpetto**

### CODE REVIEW

File: oregon-b/snippets-stage/deploy.yaml [hunk 1]
```diff
@@ -38,278 +37,157 @@ spec:
...
             secretKeyRef:
               key: cache-url
-              name: snippets-stage-v191-env
+              name: snippets-stage-secrets
         - name: CSP_REPORT_ENABLE
-          valueFrom:
-            secretKeyRef:
-              key: csp-report-enable
-              name: snippets-stage-v191-env
+          value: "True"
```
Comment (jgmize) [inline oregon-b/snippets-stage/deploy.yaml:71]: This is currently False in stage

File: oregon-b/snippets-stage/deploy.yaml [hunk 2]
```diff
@@ -38,278 +37,157 @@ spec:
...
-            secretKeyRef:
-              key: secure-content-type-nosniff
-              name: snippets-stage-v191-env
+          value: "True"
         - name: SECURE_HSTS_SECONDS
-          valueFrom:
-            secretKeyRef:
-              key: secure-hsts-seconds
-              name: snippets-stage-v191-env
+          value: "3600"
```
Comment (jgmize) [inline oregon-b/snippets-stage/deploy.yaml:151]: This is the current value in stage, but should probably be increased to 31536000 to match prod

File: oregon-b/snippets-stage/deploy.yaml [hunk 3]
```diff
@@ -38,278 +37,157 @@ spec:
...
-            secretKeyRef:
-              key: enable-admin
-              name: snippets-stage-v191-env
+          value: "True"
         - name: ENFORCE_HOST
-          valueFrom:
-            secretKeyRef:
-              key: enforce-host
-              name: snippets-stage-v191-env
+          value: "snippets.allizom.org"
```
Comment (jgmize) [inline oregon-b/snippets-stage/deploy.yaml:99]: As we discussed earlier on zoom, this is the current setting in stage, but neither dev nor prod have this setting and I'm not sure if this has any effect given that `ALLOWED_HOSTS=*`. This appears to be a leftover from testing and is likely something we should remove-- @glogiotatidis can you confirm?

**Review by jgmize — APPROVED** [review]
None of my comments are blockers for this PR; we can follow up on them shortly.

**Comment by jpetto** [comment]
mozmeao/infra#986

**Pull request merged**